modules/apps/transfer/ibc_module_test.go

```diff
@@ -239,3 +240,84 @@ func (suite *TransferTestSuite) TestOnChanOpenAck() {
 		})
 	}
 }
+
+func (suite *TransferTestSuite) TestOnChanUpgradeInit() {
+	var (
+		path *ibctesting.Path
+	)
+
+	testCases := []struct {
+		name     string
+		malleate func()
+		expError error
+	}{
+		{
+			"success",
+			func() {},
+			nil,
```

Comment:
A successful happy path upgrade for a standalone transfer app is swapping out the underlying connection. 

I can add an in-line comment if people think its useful

Reply:
I think test is readable, comment def wouldn't hurt